README.md

```diff
@@ -18,24 +18,31 @@ Install the `RectorLaravel` package as dependency:
 composer require driftingly/rector-laravel --dev
 ```
 
-## Use Sets
+## Automate Laravel Upgrades
 
-To add a set to your config, use `RectorLaravel\Set\LaravelSetList` class and pick one of the constants:
+To add a set to your config, use `RectorLaravel\Set\LaravelLevelSetList` and pick the constant that matches your target version.
+Sets for higher versions include sets for lower versions.
 
 ```php
-<?php
+return Rector\Config\RectorConfig::configure()
+    ->withSets([
+        RectorLaravel\Set\LaravelLevelSetList::LARAVEL_110,
```

Comment:
The `LaravelLevelSetList` class only has constants like `UP_TO_LARAVEL_*`, so change the line to `RectorLaravel\Set\LaravelLevelSetList::UP_TO_LARAVEL_110` please.

Also, keep the rest of the code example the same, since it provides context and shows the `rector.php` file is very minimal. So bring back this part please:
```php
<?php

declare(strict_types=1);

use Rector\Config\RectorConfig;
use RectorLaravel\Set\LaravelSetList;

return RectorConfig::configure()
```

Reply:
I readded a shortened PHP opening tag and imports and fixed the constant.